doc/tutorials/active_dendrite/nestml_active_dendrite_tutorial.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# NESTML active dendrite tutorial\n",
+    "\n",
+    "In this tutorial, we create a neuron model with an active dendritic compartment, that will fire a dendritic spike if the total synaptic current exceeds a threshold.\n",
```

Comment:
1. I think we should change the term **dendritic spike** to **dendritic action potential (dAP)** throughout all the tutorial. The dendritic activity we are modeling is not spiky but rather has long-lasting temporal dynamics  (e.g. current plateaux)
2. add that these dAPs mimic NMDA dynamics in their shape and temporal duration, including some references
3. add that the model we are building in this tutorial constitutes an adapted version of the neuron model introduced by Memmesheimer et al. (2012) and Jahnke et al. (2012).

Memmesheimer, R. M., & Timme, M. (2012). Non-additive coupling enables propagation of synchronous spiking activity in purely random networks. PLoS Comput Biol, 8(4), e1002384.

Jahnke, S., Timme, M., & Memmesheimer, R. M. (2012). Guiding synchrony through random networks. Physical Review X, 2(4), 041016.

Reply:
1. done
2. I found Srdjan D. Antic, Wen-Liang Zhou, Anna R. Moore, Shaina M. Short, and Katerina D. Ikonomu (2010). The Decade of the Dendritic NMDA Spike. J Neurosci Res. Nov 1, 88(14). This includes a good illustration of the plateau potential.
3. done